src/components/NewDraftWizard/WizardComponents/WizardDraftSelections.js

```diff
@@ -0,0 +1,197 @@
+//@flow
+import React from "react"
+
+import Button from "@material-ui/core/Button"
+import Checkbox from "@material-ui/core/Checkbox"
+import FormControl from "@material-ui/core/FormControl"
+import MuiFormControlLabel from "@material-ui/core/FormControlLabel"
+import FormGroup from "@material-ui/core/FormGroup"
+import FormLabel from "@material-ui/core/FormLabel"
+import ListItemText from "@material-ui/core/ListItemText"
+import { makeStyles, withStyles } from "@material-ui/core/styles"
+import { useForm, FormProvider, useFormContext } from "react-hook-form"
+import { useSelector, useDispatch } from "react-redux"
+import { useHistory } from "react-router-dom"
+
+import { ObjectSubmissionTypes, ObjectStatus } from "constants/wizardObject"
+import { WizardStatus } from "constants/wizardStatus"
+import { setCurrentObject } from "features/wizardCurrentObjectSlice"
+import { updateStatus } from "features/wizardStatusMessageSlice"
+import { setSubmissionType } from "features/wizardSubmissionTypeSlice"
+import draftAPIService from "services/draftAPI"
+import type { ObjectInsideFolderWithTags } from "types"
+import { getItemPrimaryText } from "utils"
+
+const useStyles = makeStyles(theme => ({
+  formComponent: {
+    margin: theme.spacing(2),
+    padding: 0,
+    overflow: "scroll",
```

Comment:
![image](https://user-images.githubusercontent.com/47524/114132899-dd382180-990d-11eb-94c0-ffee8e4adfb1.png)
There is some weird formatting I think we can change this to `auto` and it will be solved, but I have not tested with more items
```suggestion
    overflow: "auto",
```